.azure-pipelines-steps.yml

```diff
@@ -0,0 +1,52 @@
+# Build steps common to all platforms
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: $(node_version)
+  displayName: 'Install Node.js'
+
+- task: Npm@1
+  inputs:
+    command: install
+  displayName: 'Install dependencies'
+
+- bash: |
+    node_modules/.bin/mocha test --reporter mocha-junit-reporter --reporter-options mochaFile=./test-results.xml
+  displayName: 'Run tests'
+
+- task: PublishTestResults@2
+  inputs:
+    testResultsFiles: '**/test-results.xml'
+    testRunTitle: $(os_name) Node $(node_version)
+  displayName: 'Publishing the test results'
+  condition: always()
+
+- bash: |
+      export DISPLAY=:99.0
```

Comment:
Since the DISPLAY variable is only relevant on Linux, I would elevate it to a variable on the Linux job. I would then use it when you call Xvfb: 

```
/usr/bin/Xvfb ${DISPLAY} -ac >> /tmp/xvfb.out 2>&1 &
```

This avoids a problem down the road when someone changes the value of DISPLAY, but forgets to change the Xvfb call.